### PR TITLE
V6/add user error message

### PR DESCRIPTION
**Pull request by hsurf22**
### Devop

* \[x] Updated CHANGELOG.md
* \[x] Add PR label
   - Add user error message

Screenshot from 2021-05-24 19-55-45
Screenshot from 2021-05-24 19-55-54

### CODE REVIEW

File: src/app.vue [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
   <v-app class="walletBg">
+    <msg />
     <module-decision-tree />
     <router-view />
     <module-toast />
   </v-app>
 </template>
 
 <script>
+import msg from '@/core/components/AppErrorMsg';
```
Comment (olgakup) [inline src/app.vue:11]: change msg to AppErrorMsg

File: src/core/components/AppErrorMsg.vue [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    </v-img>
+
+    <div style="background-color: white" class="py-6">
+      <div class="px-6">
+        <div>
+          Something goes wrong, would you want to inform to #MEWteam about this
+          error?
+        </div>
+        <div class="d-flex align-center justify-center my-7">
+          <mew-button
```
Comment (olgakup) [inline src/core/components/AppErrorMsg.vue:18]: make this button full width

File: src/core/components/AppErrorMsg.vue [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+            btn-style="outline"
+            btn-size="xlarge"
+            title="Ok"
+            class="mx-1"
+          />
+          <mew-button btn-size="xlarge" title="Send" class="mx-1" />
```
Comment (olgakup) [inline src/core/components/AppErrorMsg.vue:24]: make this button full width

**Review by olgakup — CHANGES_REQUESTED** [review]

**Comment by olgakup** [comment]
The padding between the buttons and error message looks smaller in the designs then in implementation
Screen Shot 2021-05-25 at 12 03 23 PM

File: src/core/components/AppErrorMsg.vue (hunk 2 above)
Comment (hsurf22) [reply src/core/components/AppErrorMsg.vue]: fixed

File: src/core/components/AppErrorMsg.vue (hunk 3 above)
Comment (hsurf22) [reply src/core/components/AppErrorMsg.vue]: fixed

**Review by olgakup — APPROVED** [review]

**Comment by olgakup** [comment]
@hsurf22 can you fix the build? and then merge this?

**Pull request merged**